docs/docs/events.md

```diff
@@ -133,17 +133,45 @@ commandCompletedListener.remove();
 |**completionTime**|Timestamp when the command, and consecutive animations, completed.|
 
 ## registerModalDismissedListener
-Invoked when modal dismissed.
+
+Called each time this component (Modal) is dismissed. 
+
+```js
+class MyComponent extends Component {
+
+  componentDidMount() {
+    this.navigationEventListener = Navigation.events().bindComponent(this);
+  }
+
+  componentWillUnmount() {
+    // Not mandatory
+    if (this.navigationEventListener) {
+      this.navigationEventListener.remove();
+    }
+  }
+
+  modalDismissed({ componentId, componentName, modalsDismissed }) {
+
+  }
+}
+```
+
+This event can be observed globally as well:
 
 ```js
 // Subscribe
-const modalDismissedListener = Navigation.events().registerModalDismissedListener(({ componentId, modalsDismissed }) => {
+const modalDismissedListener = Navigation.events().registerModalDismissedListener(({ componentId, componentName, modalsDismissed }) => {
 
 });
 ...
 // Unsubscribe
 modalDismissedListener.remove();
 ```
+|       Parameter         | Description |
+|:--------------------:|:-----|
+|**componentId**| Id of the disappearing component|
+|**componentName**|Registered name used when registering the component with `Navigation.registerComponent()`|
```

Comment:
Lets change the description to "Registered name of the disappearing component". WDYT?

Reply:
Oops. Bad copy and paste skill :( Fixed.